src/notation/qml/MuseScore/NotationScene/PercussionPanel.qml

```diff
@@ -0,0 +1,187 @@
+/*
+ * SPDX-License-Identifier: GPL-3.0-only
+ * MuseScore-Studio-CLA-applies
+ *
+ * MuseScore Studio
+ * Music Composition & Notation
+ *
+ * Copyright (C) 2024 MuseScore Limited
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+import QtQuick 2.15
+
+import Muse.Ui 1.0
+import Muse.UiComponents 1.0
+import MuseScore.NotationScene 1.0
+
+import "internal"
+
+Item {
+    id: root
+
+    property NavigationSection navigationSection: null
+    property int contentNavigationPanelOrderStart: 1
+
+    anchors.fill: parent
+
+    //TODO: #22050 needed for this
+    /*
+    property Component toolbarComponent: PercussionPanelToolBar {
+        navigation.section: root.navigationSection
+        navigation.order: root.contentNavigationPanelOrderStart
+
+        model: percussionPanelModel
+
+        panelWidth: root.width
+    }
+    */
+
+    PercussionPanelModel {
+        id: percModel
+    }
+
+    // We'll use "dummy areas" until #22050 is completed
+    Rectangle {
+        id: dummyToolBarArea
+
+        anchors.top: parent.top
+        anchors.left: parent.left
+        anchors.right: parent.right
+
+        height: 35
+
+        color: ui.theme.backgroundSecondaryColor
+
+        PercussionPanelToolBar {
+            navigation.section: root.navigationSection
+            navigation.order: root.contentNavigationPanelOrderStart
+
+            model: percModel
+
+            panelWidth: root.width
+        }
+    }
+
+    Item {
+        id: dummyPanelArea
+
+        anchors.top: dummyToolBarArea.bottom
+        anchors.left: parent.left
+        anchors.right: parent.right
+        anchors.bottom: parent.bottom
+        anchors.margins: 10
+
+        clip: true
+
+        StyledGridView {
+            id: padGrid
+
+            readonly property int numColumns: model.numColumns
+            readonly property int spacing: cellWidth / 6
+
+            property int numRows: Math.floor(model.rowCount() / numColumns) + 1
+
+            anchors.centerIn: parent
+
+            height: parent.height
+            width: cellWidth * numColumns
+
+            clip: false
+
+            cellWidth: 125
```

Comment:
126 or 128 (we should avoid numbers that are not multiples of 2 or 4)

Reply:
I actually messed up here - it should be `100 + padGrid.spacing` (spacing being 20)